src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
```

Comment:
Maybe can include the '[' and ']' chars in the strings so that you don't need to that when printing the list.